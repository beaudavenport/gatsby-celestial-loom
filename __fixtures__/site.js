export default {
  site: {
    siteMetadata: {
      title: 'The Celestial Loom',
    },
  },
};
