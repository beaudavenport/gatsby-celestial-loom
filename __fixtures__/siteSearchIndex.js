export default {
  siteSearchIndex: {
    index: null,
  },
};
