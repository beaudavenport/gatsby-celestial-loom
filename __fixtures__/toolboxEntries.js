export default {
  allMarkdownRemark: {
    edges: [
      {
        node: {
          id: '58aa600c-e761-5ebc-b285-eb6d59ecc4ac',
          frontmatter: { title: 'Aries', type: 'toolbox', toolboxType: 'Signs' },
          excerpt: 'Key Phrase:  I AM Basic Nature:  Pioneer and Warrior Keywords:  courageous, independent, assertive, spontaneous, head of cosmic man, self…',
          fields: { slug: '/toolbox/aries/' },
        },
      }, {
        node: {
          id: 'e9866973-380c-54ee-96c4-6a44b09dc958',
          frontmatter: { title: 'First House', type: 'toolbox', toolboxType: 'Houses' },
          excerpt: 'I AM -  The appearance, the basic mood in life, personality, vital energy, awareness of life, physical constitution, vitality, expression of…',
          fields: { slug: '/toolbox/first-house/' },
        },
      }, {
        node: {
          id: 'c96e763f-d670-5bef-a05c-3792d86a0a60',
          frontmatter: { title: 'Sun', type: 'toolbox', toolboxType: 'Planets' },
          excerpt: 'Everything in the Solar System revolves around the Sun. Growth, self-expression, outer-world activities Positive energy in any endeavor Tap…',
          fields: { slug: '/toolbox/sun/' },
        },
      }, {
        node: {
          id: 'f1fc71be-24db-5613-991a-0fdc62d5e03b',
          frontmatter: { title: 'Moon', type: 'toolbox', toolboxType: 'Planets' },
          excerpt: 'In our Solar System, the Moon reflects the light. Fundamental energy tied to emotions and intuition Celestial power tied to the Goddess Moon…',
          fields: { slug: '/toolbox/moon/' },
        },
      }, {
        node: {
          id: 'a8eabb89-ed90-5b1b-a942-80aeee38994f',
          frontmatter: { title: 'Second House', type: 'toolbox', toolboxType: 'Houses' },
          excerpt: 'I HAVE  - Assets, foundation of material, intellectual, and emotional security, wealth, possessions, finances, social background, money…',
          fields: { slug: '/toolbox/second-house/' },
        },
      }, {
        node: {
          id: '693f6b38-1f10-5555-b6f8-43cbaac2d26e',
          frontmatter: { title: 'Taurus', type: 'toolbox', toolboxType: 'Signs' },
          excerpt: 'Key Phrase:  I HAVE Basic Nature:  Builder and Accumulator Keywords:  courageous, independent, assertive, spontaneous, head of cosmic man…',
          fields: { slug: '/toolbox/taurus/' },
        },
      }, {
        node: {
          id: 'bbd334c8-9405-5f26-98d9-9617ebf82874',
          frontmatter: { title: 'Gemini', type: 'toolbox', toolboxType: 'Signs' },
          excerpt: 'Key Phrase:  I THINK Basic Nature:  Communicator and Threshold Walker Keywords:  intellectual, communicative, curious, versatile, lower neck…',
          fields: { slug: '/toolbox/gemini/' },
        },
      }, {
        node: {
          id: '80acb630-2187-5267-a599-c4359b0d68a3',
          frontmatter: { title: 'Mercury', type: 'toolbox', toolboxType: 'Planets' },
          excerpt: 'Mercury is the messenger of the Gods. Rules speech and thinking Tap energy when you use affirmations, sigils, hand gestures, and thought…',
          fields: { slug: '/toolbox/mercury/' },
        },
      },
    ],
  },
};
