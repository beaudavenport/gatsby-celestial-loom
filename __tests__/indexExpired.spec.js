import { StaticQuery, useStaticQuery } from 'gatsby';
import React from 'react';

import renderer from 'react-test-renderer';

import IndexPage from '../src/pages/index';
import indexPageExpiredEntries from '../__fixtures__/indexPageExpiredEntries';
import sidebarEntries from '../__fixtures__/sidebarEntries';
import site from '../__fixtures__/site';
import siteSearchIndex from '../__fixtures__/siteSearchIndex';
import toolboxEntries from '../__fixtures__/toolboxEntries';


beforeEach(() => {
  global.window.matchMedia = jest.fn().mockImplementation(query => ({
    matches: false,
    media: query,
    onchange: null,
    addListener: jest.fn(),
    removeListener: jest.fn(),
  }));

  StaticQuery.mockImplementation(({ render }) => render({
    ...site,
    ...siteSearchIndex,
    ...sidebarEntries,
  }));

  useStaticQuery.mockReturnValueOnce(toolboxEntries);
});

describe('Index Expired Page', () => {
  it('should render correctly with posts, expired events, and services', () => {
    const tree = renderer
      .create(<IndexPage data={indexPageExpiredEntries} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
