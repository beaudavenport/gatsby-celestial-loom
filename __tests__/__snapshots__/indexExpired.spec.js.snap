// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Expired Page should render correctly with posts, expired events, and services 1`] = `
<div>
  <header
    className="md-paper md-paper--2 md-toolbar md-background--primary md-toolbar--text-white md-toolbar--fixed"
    style={
      Object {
        "backgroundColor": "rgb(47, 47, 47)",
      }
    }
  >
    <button
      className="md-btn md-btn--icon md-pointer--hover md-inline-block md-btn--toolbar md-toolbar--action-left"
      disabled={false}
      onClick={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      type="button"
    >
      <div
        className="md-ink-container"
      />
      <i
        className="md-icon material-icons md-text--inherit"
      >
        menu
      </i>
    </button>
    <h2
      className="md-title md-title--toolbar"
      style={
        Object {
          "fontFamily": "Berkshire Swash",
          "fontStyle": "italic",
        }
      }
    >
      Home
    </h2>
    <div
      className="md-cell--right md-toolbar--action-right"
    >
      <div
        className="md-btn--toolbar header-icons-container"
      >
        <div
          className="header-search-container"
        >
          <button
            className="md-btn md-pointer--hover md-text md-inline-block"
            onClick={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
            style={
              Object {
                "alignItems": "center",
                "display": "flex",
              }
            }
            type="button"
          >
            <div
              className="md-ink-container"
            />
            <div
              className="md-badge-container md-inline-block"
            >
              <i
                className="md-icon material-icons"
                style={
                  Object {
                    "fontSize": "2rem",
                  }
                }
              >
                shopping_cart
              </i>
            </div>
          </button>
        </div>
        <div
          className="header-search-container"
        >
          <button
            className="md-btn md-pointer--hover md-text md-inline-block"
            onClick={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
            style={
              Object {
                "alignItems": "center",
                "display": "flex",
              }
            }
            type="button"
          >
            <div
              className="md-ink-container"
            />
            <i
              className="md-icon material-icons"
              style={
                Object {
                  "fontSize": "2rem",
                }
              }
            >
              search
            </i>
          </button>
        </div>
      </div>
    </div>
  </header>
  <main
    className="md-navigation-drawer-content md-navigation-drawer-content--inactive md-transition--acceleration md-toolbar-relative"
    id="main-content"
    tabIndex={-1}
  >
    <div
      className="md-paper md-paper--0 jumbotron--bg"
      style={
        Object {
          "backgroundImage": "url(File contents (usually images))",
        }
      }
    >
      <div
        className="jumbotron--container"
      >
        <div
          style={
            Object {
              "alignItems": "center",
              "display": "flex",
              "marginBottom": 30,
            }
          }
        >
          <div>
            <h1
              className="jumbotron--title"
            >
              The Celestial Loom
            </h1>
            <p
              className="jumbotron--subtitle"
            >
              Astrological Services
            </p>
          </div>
        </div>
        <div
          className="jumbotron--subtext--container"
        >
          <p
            className="jumbotron--subtext"
          >
            Nikki Davenport, Astrological Consultant, weaves timeless wisdom with modern service.
          </p>
          <div
            style={
              Object {
                "textAlign": "right",
              }
            }
          >
            <div
              className="link--container link--container__right"
            >
              <a
                href="/getting-started"
                style={
                  Object {
                    "textDecoration": "none",
                  }
                }
              >
                <div
                  className="touchable-link "
                  style={Object {}}
                >
                  <div
                    className="md-ink-container"
                  />
                  <div
                    className="flex-center"
                  >
                    <p
                      className="back-title"
                      style={
                        Object {
                          "fontSize": "1.5rem",
                        }
                      }
                    >
                      Get Started
                    </p>
                    <i
                      className="md-icon material-icons"
                      style={
                        Object {
                          "color": "#ec6602",
                          "fontSize": "2rem",
                          "marginLeft": 5,
                        }
                      }
                    >
                      arrow_forward
                    </i>
                  </div>
                </div>
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="md-grid md-grid--no-spacing"
      style={
        Object {
          "maxWidth": 1100,
        }
      }
    >
      <div
        className="md-cell md-cell--8 main-contents"
      >
        <div
          className="md-grid"
          style={
            Object {
              "borderBottom": "1px solid rgba(15,70,100,.2)",
              "marginBottom": 20,
            }
          }
        >
          <div
            className="md-cell md-cell--12"
          >
            <p
              className="subheader"
              style={
                Object {
                  "fontSize": "1.5rem",
                  "lineHeight": "1.5rem",
                }
              }
            >
              Featured Post
            </p>
          </div>
          <div
            className="md-cell md-cell--12"
            style={
              Object {
                "marginBottom": 10,
              }
            }
          >
            <div>
              <a
                href="/blog/norwac-i-am-on-my-way/"
                style={
                  Object {
                    "textDecoration": "none",
                  }
                }
              >
                <div
                  className="touchable-link "
                  style={
                    Object {
                      "padding": 10,
                    }
                  }
                >
                  <div
                    className="md-ink-container"
                  />
                  <div
                    className="featured-card--container"
                  >
                    <img
                      alt="Preview of featured post"
                      className="featured-card--image"
                      src="/assets/cancer-norwac.jpg"
                    />
                    
                  </div>
                  <div
                    style={
                      Object {
                        "padding": 16,
                      }
                    }
                  >
                    <h2
                      className="title"
                    >
                      NORWAC - I AM ON MY WAY
                    </h2>
                    <h3
                      className="featured-card--publish-date"
                    >
                      May 23, 2019
                    </h3>
                  </div>
                  <section
                    className="md-card-text"
                    style={
                      Object {
                        "padding": 16,
                      }
                    }
                  >
                    <div
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "<p>I'm on my way to NORWAC - an astrological convention in Seattle, WA.  It has been almost 20 years since I was at a big conference and I am bursting with delight.  The action begins on Friday, May 24 at 1:00 PM and the event chart explodes withâ€¦</p>",
                        }
                      }
                    />
                    <div
                      className="chevron-text"
                    >
                      <p
                        className="chevron-text--text"
                      >
                        Read more
                      </p>
                      <i
                        className="md-icon material-icons chevron-text--chevron"
                      >
                        chevron_right
                      </i>
                    </div>
                  </section>
                </div>
              </a>
              <div
                className="padded-section"
              >
                <p
                  className="subheader"
                >
                  Mentioned in this post:
                </p>
                <div
                  className="related-items--container"
                >
                  <button
                    className="md-chip md-chip--avatar"
                    onClick={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                    style={
                      Object {
                        "cursor": "pointer",
                        "margin": 10,
                      }
                    }
                    type="button"
                  >
                    <div
                      className="md-inline-block md-avatar md-avatar--default md-avatar--icon-sized"
                    >
                      <div
                        className="md-avatar-content"
                      >
                        <span
                          style={
                            Object {
                              "fontFamily": "AstroGadget",
                            }
                          }
                        >
                          c
                        </span>
                      </div>
                    </div>
                    <span
                      className="md-chip-text"
                    >
                      Gemini
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="md-grid"
          style={
            Object {
              "borderBottom": "1px solid rgba(15,70,100,.2)",
              "marginBottom": 20,
            }
          }
        >
          <div
            className="md-cell md-cell--12"
          >
            <div
              style={
                Object {
                  "alignItems": "baseline",
                  "display": "flex",
                  "justifyContent": "space-between",
                }
              }
            >
              <p
                className="subheader"
                style={
                  Object {
                    "fontSize": "1.5rem",
                    "lineHeight": "1.5rem",
                  }
                }
              >
                Previous Posts
              </p>
              <div
                className="link--container false"
              >
                <a
                  href="/posts"
                  style={
                    Object {
                      "textDecoration": "none",
                    }
                  }
                >
                  <div
                    className="touchable-link "
                    style={Object {}}
                  >
                    <div
                      className="md-ink-container"
                    />
                    <div
                      className="flex-center"
                    >
                      <p
                        className="back-title"
                      >
                        All Posts
                      </p>
                      <i
                        className="md-icon material-icons"
                        style={
                          Object {
                            "color": "#ec6602",
                            "fontSize": "2rem",
                            "marginLeft": 5,
                          }
                        }
                      >
                        arrow_forward
                      </i>
                    </div>
                  </div>
                </a>
              </div>
            </div>
          </div>
          <div
            className="md-cell md-cell--6 md-cell--4-tablet"
            style={
              Object {
                "padding": 10,
              }
            }
          >
            <a
              href="/blog/the-pleiades/"
              style={
                Object {
                  "textDecoration": "none",
                }
              }
            >
              <div
                className="touchable-link thumbnail-card"
                style={Object {}}
              >
                <div
                  className="md-ink-container"
                />
                <div
                  className="thumbnail-image"
                  style={
                    Object {
                      "backgroundImage": "url(/assets/pleiades.jpg)",
                    }
                  }
                />
                <div
                  className="thumbnail-text"
                >
                  <p
                    className="thumbnail-text--title"
                  >
                    The Pleiades
                  </p>
                  <p
                    className="thumbnail-text--subtitle"
                  >
                    Posted on:
                     
                    <span
                      style={
                        Object {
                          "color": "rgba(0, 0, 0, 0.87)",
                          "fontWeight": "bold",
                        }
                      }
                    >
                      May 21, 2019
                    </span>
                  </p>
                </div>
              </div>
            </a>
          </div>
        </div>
        <div
          className="md-grid"
          style={
            Object {
              "borderBottom": "1px solid rgba(15,70,100,.2)",
              "marginBottom": 20,
            }
          }
        >
          <div
            className="md-cell md-cell--12"
          >
            <div
              style={
                Object {
                  "alignItems": "baseline",
                  "display": "flex",
                  "justifyContent": "space-between",
                }
              }
            >
              <p
                className="subheader"
                style={
                  Object {
                    "fontSize": "1.5rem",
                    "lineHeight": "1.5rem",
                  }
                }
              >
                Featured Service
              </p>
              <div
                className="link--container false"
              >
                <a
                  href="/services"
                  style={
                    Object {
                      "textDecoration": "none",
                    }
                  }
                >
                  <div
                    className="touchable-link "
                    style={Object {}}
                  >
                    <div
                      className="md-ink-container"
                    />
                    <div
                      className="flex-center"
                    >
                      <p
                        className="back-title"
                      >
                        All Services
                      </p>
                      <i
                        className="md-icon material-icons"
                        style={
                          Object {
                            "color": "#ec6602",
                            "fontSize": "2rem",
                            "marginLeft": 5,
                          }
                        }
                      >
                        arrow_forward
                      </i>
                    </div>
                  </div>
                </a>
              </div>
            </div>
          </div>
          <div
            className="md-cell md-cell--12"
          >
            <div
              className="md-paper md-paper--1 md-card md-background--card"
              onMouseLeave={[Function]}
              onMouseOver={[Function]}
              onTouchStart={[Function]}
            >
              <a
                href="/services/natal-chart/"
                style={
                  Object {
                    "textDecoration": "none",
                  }
                }
              >
                <div
                  className="touchable-link "
                  style={Object {}}
                >
                  <div
                    className="md-ink-container"
                  />
                  <div
                    className="origin--header--bg"
                    style={
                      Object {
                        "backgroundImage": "url(File contents (usually images))",
                      }
                    }
                  >
                    <div
                      className="origin--header--overlay"
                      style={
                        Object {
                          "background": "rgba(39,62,84,0.65)",
                        }
                      }
                    >
                      <p
                        className="origin--header--title"
                      >
                        Western
                      </p>
                    </div>
                  </div>
                  <section
                    className="md-card-text"
                  >
                    <h2
                      className="title"
                    >
                      Natal Chart
                    </h2>
                    <div
                      className="service--excerpt"
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "<p>This chart is the perfect introduction to understanding your Astrological dynamics. With an in-depth look into the planetary energies at work since your life began, you can make informed decisions about the world around you and the future that youâ€¦</p>",
                        }
                      }
                    />
                    <hr
                      className="md-divider"
                    />
                    <div
                      className="service--prices--container"
                    >
                      <div>
                        <p
                          className="service--price--description"
                        >
                          Starting at
                        </p>
                        <p
                          className="service--price--price"
                        >
                          $140.00
                        </p>
                      </div>
                      <div
                        className="chevron-text"
                      >
                        <p
                          className="chevron-text--text"
                        >
                          Order Now
                        </p>
                        <i
                          className="md-icon material-icons chevron-text--chevron"
                        >
                          chevron_right
                        </i>
                      </div>
                    </div>
                  </section>
                </div>
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        className="md-cell md-cell--4 md-cell--8-tablet sidebar-contents"
      >
        <div
          className="md-grid md-grid--no-spacing"
        >
          <div
            className="md-cell md-cell--12"
          >
            <div
              className="sidebar-header"
            >
              <h5
                className="sidebar-header--title"
              >
                About Nikki
              </h5>
            </div>
          </div>
          <div
            className="md-cell md-cell--12 md-cell--4-tablet about-nikki--card"
          >
            <div>
              <div
                className="about-nikki--container"
              >
                <section
                  className="md-media md-media--1-1 about-nikki--avatar"
                >
                  <img
                    alt="at da club"
                    src="File contents (usually images)"
                  />
                </section>
              </div>
              <section
                className="md-card-text flex-column"
              >
                <p
                  className="about-nikki--text"
                >
                  Nikki Davenport, astrological consultant, has been a professional astrologer for over 35 years.
                </p>
                <a
                  className="md-btn md-btn--flat md-btn--text md-pointer--hover md-text--theme-primary md-ink--primary md-inline-block about-nikki--button"
                  href="/about-nikki"
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  type={null}
                >
                  <div
                    className="md-ink-container"
                  />
                  About Nikki
                </a>
              </section>
            </div>
          </div>
          <div
            className="md-cell md-cell--12"
          >
            <div
              className="sidebar-header"
            >
              <h5
                className="sidebar-header--title"
              >
                Stay Connected
              </h5>
            </div>
          </div>
          <div
            className="md-cell md-cell--12 md-cell--4-tablet"
          >
            <div
              style={
                Object {
                  "paddingRight": 1,
                }
              }
            >
              <a
                className="twitter-timeline"
                data-tweet-limit="1"
                href="https://twitter.com/nikiastro?ref_src=twsrc%5Etfw"
              >
                Tweets by nikiastro
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="md-grid footer--grid"
    >
      <div
        className="md-cell md-cell--6 md-cell--8-tablet"
      >
        <div
          className="flex-center"
        >
          <div
            style={
              Object {
                "marginRight": 10,
              }
            }
          >
            <img
              alt="Celestial Loom Logo"
              src="File contents (usually images)"
              style={
                Object {
                  "width": 50,
                }
              }
            />
          </div>
          <h3
            className="footer--title"
          >
            The Celestial Loom
          </h3>
        </div>
        <div
          style={
            Object {
              "padding": 10,
            }
          }
        >
          <p
            className="footer--text"
          >
            Â©
             Copyright 2019, Nikki Davenport.
          </p>
          <p
            className="footer--text"
          >
            Cover photography by Photo by Anastasia Dulgier on Unsplash.
          </p>
          <p
            className="footer--text"
          >
            Built with Gatsby and Netflify CMS.
          </p>
        </div>
      </div>
      <div
        className="md-cell md-cell--6 md-cell--8-tablet"
      >
        <div
          className="md-grid"
        >
          <div
            className="md-cell md-cell--6 md-cell--4-tablet"
          >
            <p
              className="subheader"
            >
              Links
            </p>
            <ul
              className="footer--list"
            >
              <li
                className="footer--line-item"
              >
                <a
                  href="/home/"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                >
                  Home
                </a>
              </li>
              <li
                className="footer--line-item"
              >
                <a
                  href="/about-nikki/"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                >
                  About Nikki
                </a>
              </li>
              <li
                className="footer--line-item"
              >
                <a
                  href="/getting-started/"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                >
                  Getting Started
                </a>
              </li>
              <li
                className="footer--line-item"
              >
                <a
                  href="/posts/"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                >
                  Posts
                </a>
              </li>
              <li
                className="footer--line-item"
              >
                <a
                  href="/events/"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                >
                  Events
                </a>
              </li>
              <li
                className="footer--line-item"
              >
                <a
                  href="/services/"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                >
                  Services
                </a>
              </li>
              <li
                className="footer--line-item"
              >
                <a
                  href="/toolbox/"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                >
                  Astro Toolbox
                </a>
              </li>
            </ul>
          </div>
          <div
            className="md-cell md-cell--6 md-cell--4-tablet"
          >
            <p
              className="subheader"
            >
              Contact
            </p>
            <ul
              className="footer--list"
            >
              <li>
                <div
                  className="footer--link"
                >
                  <i
                    className="md-icon material-icons"
                    style={
                      Object {
                        "color": "white",
                      }
                    }
                  >
                    email
                  </i>
                  <a
                    className="footer--link__link"
                    href="mailto: nikiastro@att.net"
                  >
                    Email Nikki
                  </a>
                </div>
              </li>
              <li>
                <div
                  className="footer--link"
                >
                  <svg
                    fill="white"
                    height={20}
                    viewBox="0 0 24 24"
                    width={20}
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M12 0c-6.627 0-12 5.373-12 12s5.373 12 12 12 12-5.373 12-12-5.373-12-12-12zm3 8h-1.35c-.538 0-.65.221-.65.778v1.222h2l-.209 2h-1.791v7h-3v-7h-2v-2h2v-2.308c0-1.769.931-2.692 3.029-2.692h1.971v3z"
                    />
                  </svg>
                  <a
                    className="footer--link__link"
                    href="https://www.facebook.com/celestialloom/"
                  >
                    Facebook
                  </a>
                </div>
              </li>
              <li>
                <div
                  className="footer--link"
                >
                  <svg
                    fill="white"
                    height={20}
                    viewBox="0 0 24 24"
                    width={20}
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M12 0c-6.627 0-12 5.373-12 12s5.373 12 12 12 12-5.373 12-12-5.373-12-12-12zm6.066 9.645c.183 4.04-2.83 8.544-8.164 8.544-1.622 0-3.131-.476-4.402-1.291 1.524.18 3.045-.244 4.252-1.189-1.256-.023-2.317-.854-2.684-1.995.451.086.895.061 1.298-.049-1.381-.278-2.335-1.522-2.304-2.853.388.215.83.344 1.301.359-1.279-.855-1.641-2.544-.889-3.835 1.416 1.738 3.533 2.881 5.92 3.001-.419-1.796.944-3.527 2.799-3.527.825 0 1.572.349 2.096.907.654-.128 1.27-.368 1.824-.697-.215.671-.67 1.233-1.263 1.589.581-.07 1.135-.224 1.649-.453-.384.578-.87 1.084-1.433 1.489z"
                    />
                  </svg>
                  <a
                    className="footer--link__link"
                    href="https://twitter.com/nikiastro"
                  >
                    Twitter
                  </a>
                </div>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </main>
</div>
`;
